app/src/main/res/values/strings.xml

```diff
@@ -624,5 +624,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_tracktype_grade5">Soft</string>
     <string name="quest_building_underground_name_title">Is %s completely underground?</string>
     <string name="quest_building_underground_title">Is this building completely underground?</string>
+    <string name="quest_traffic_signals_button_title">Do these traffic lights have a button?</string>
```

Comment:
"to request a green light" (I am not really happy with that but I have no better ideas). Or maybe add a photo with the button?

Reply:
How about ["push-button"](https://en.wikipedia.org/wiki/Push-button) instead of "button"?

"Do these traffic lights have a push-button that can increase pedestrian priority?"? (Admittedly nowhere near [up goer 5 friendly](http://splasho.com/upgoer5/).)

Reply:
to request green light is good. I don't know what is the difference between a push-button and a button.